packages/dds/tree/package.json

```diff
@@ -34,6 +34,7 @@
     "lint": "npm run eslint",
     "lint:fix": "npm run eslint:fix",
     "test": "npm run test:mocha",
+    "test:benchmark:report": "mocha \"dist/**/*.tests.js\" --node-option unhandled-rejections=strict,expose-gc --exit --perfMode --fgrep @Benchmark -r node_modules/@fluidframework/mocha-test-setup --reporter ../../../node_modules/@fluid-tools/benchmark/dist/MochaReporter.js --timeout 60000",
```

Comment:
generally we try not to depend on repo dir structure, and instead add dev dependencies. is there a reason we need to do that for `../../../node_modules/@fluid-tools/benchmark/dist/MochaReporter.js `

Reply:
it actually looks like you already have a dev dependency, so should just you that instead of the path

Reply:
I ran into a scenario where it didn't work, but I can't remember if I was pointing to the file starting with node_modules, or just with the package name. I think this is what happened: since Lerna hoists that dependency to the root of the repo and it doesn't create symlinks in the package's node_modules folder (I guess because it expects the regular package-resolution algorithm to eventualy find the dependency further up in the folder structure), trying to find that file "locally" fails.

It feels like pointing to `<package_name>/path/to/file` (without leading `node_modules`) should still work... I'll try that.

Reply:
Yep, prefixing the path with `node_modules/` is what seems to make it fail, with just the package name everything works fine. Made the change.